make-all.bash

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+# Build the syncthing library
+
+./make-go.bash arm
+./make-syncthing.bash arm
+
+./make-go.bash 386
+./make-syncthing.bash 386
+
+# Will need to bump target API to support this
+#./make-go.bash amd64
+#./make-syncthing.bash amd64
```

Comment:
What's the reason for this? 64 bit ARM should be supported in Lollipop. Generally, I don't really like having dead code around.